src/memote/checks/Consistency.jl

```diff
@@ -129,4 +129,56 @@ function model_is_consistent(
     termination_status(opt_model) == OPTIMAL
 end
 
+"""
+$(TYPEDSIGNATURES)
+
+Returns a list of all metabolites that aren't part of any reactions.
+"""
+function find_disconnected_metabolites(model::MetabolicModel)
+    mids = deepcopy(metabolites(model))
+    for rid in reactions(model)
+        rid_m = reaction_stoichiometry(model, rid)
+        for mid in mids
+            haskey(rid_m, mid) && deleteat!(mids, findall(x->x == mid, mids))
+        end
+    end
+    return mids
+end
+
+"""
+$(TYPEDSIGNATURES)
+
+A function to find reactions able to carry unlimited flux under default conditions.
+The function compares the fluxes of the model calculated by FVA (using flux_variability_analysis_dict)
+with the median bounds of the model (+/- treshold) and returns the fluxes which are grearter or smaller than 
+the bounds in a two seperate dictionaries.
+"""
+function unbounded_flux_in_default_medium(model::MetabolicModel, fva_result::Tuple{Any,Any}, config = memote_config)
+    tol = config.consistency.tolerance_threshold
+
```

Comment:
call `flux_variability_analysis` in the function. This way the user only passes in a `model` and an `optimizer` like the other functions

Reply:
yeah, that might be a good option. However, a few other functions also call fva, so we would have to first find them and then share the result... I think for now they should call it internally, and then in another PR find all the other ones and lift it out? Making an issue out of it, see #52.

Reply:
We were also unsure about calling it internally because FVA_dict has some optional arguments (like bounds and modifications) that the user can't modify if we call it internally with just model and optimizer as arguments. I can also try to make it so that FVA is only called internally if no fva_result is passed by the user if you want.

Reply:
forward the options where appropriate through the config file. which options do you wonder about?

Reply:
For example how I need to increase the iteration count using change_optimizer_attribute through modifications if I want to use the Tulip optimizer. I'm not sure how I would be able to tell unbounded_flux_in_default_medium that it's internal call of FVA should do that when actually using the function. But maybe I'm just missing something obvious, I'm not 100% sure what you mean by forwarding through the config?

Reply:
iteration counts are a property of a solver run, so whoever is running the external FVA should be responsible for setting these up correctly. I'd just add that to the documentation (or print a nice explanatory error message in case you see nothings, such as "FVA has no solution at points, check if the supplied solution is really OK"), deferring any complexity to the external run is OK here.

Reply:
Yes that was my thought process behind not only calling FVA in the function itself so that if the issue is with the FVA result the user has a chance to modify their call to FVA_dict themself

Reply:
yeah... we might eventually add some solver parameters to the memote config. Let's not care too much about that now, if this somehow works I guess it's okay :]

Reply:
So the final verdict is we leave it as it is now since the other functions calling FVA internally are gonna be changed anyway in #52 ? If so then everything should (hopefully) be fine now.

Reply:
yes, all OK.